src/Homepage/__snapshots__/Homepage.test.js.snap

```diff
@@ -37,6 +37,7 @@ exports[`The Homepage component renders correctly 1`] = `
     >
       Was sind Persönlichkeitsstile?
       <img
+        alt="auf/zuklappen"
```

Comment:
Is this actionable? If so I would add "icon", "button" or similar so it's clear for the visually impaired with screen readers.  ;)